sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/Azure.ApplicationModel.Configuration.Tests.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
     <PackageReference Include="BenchmarkDotNet" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Do we _really_ want to do this?

Reply:
I'm not sure. System.Text.Json APIs are so bad for what I'm trying to do. I get trying to avoid newtonsoft in shipping packages but it's so much better in tests.